openfl-tutorials/interactive_api/Director_Pytorch_Kvasir_UNET/envoy_folder/start_envoy.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 set -e
+ENVOY_NAME=$1
```

Comment:
I suggest keeping the current behaviour by adding default values for `ENVOY_NAME` and `SHARD_CONF`.